CHANGES.txt

```diff
@@ -39,6 +39,10 @@ Changes
 - Functions which accept regular expression flags now throw an error when
   invalid flags are provided.
 
+- Added support for the PSQL ParameterDescription message which allows to get
```

Comment:
I think we should point out that it's limited to select statements. (Also in the wire protocol documentation)